Funcky.Money/Visitors/MoneyBag.cs

```diff
@@ -74,7 +63,7 @@ private Money AggregateSingleMoneyBag()
         => _currencies
             .SingleOrNone() // Single or None throws an InvalidOperationException if we have more than one currency in the Bag
             .Match(
-                none: () => _emptyCurrency.Match(Money.Zero, c => Money.Zero with { Currency = c }),
+                none: () => throw new Exception("fubar"),
```

Comment:
This exception looks like a TODO

Reply:
Fixed: a55d87e3a96a74106f1db5771c360a5195cb825b

Not happy with the solution, at all, but I need a quick fix for Rapport one.
I think this Exception cannot happen, but I am not sure, adapted the text.